components/gantt/gantt-tree/columns/menu.md

```diff
@@ -0,0 +1,404 @@
+              FilterMode="@GanttFilterMode.FilterMenu"
+              ShowColumnMenu="true">
+    <GanttSettings>
+        <GanttColumnMenuSettings Lockable="false"
+                                 Reorderable="true"
+                                 FilterMode="@ColumnMenuFilterMode.None">
+        </GanttColumnMenuSettings>
+    </GanttSettings>
+    <GanttViews>
+        <GanttDayView></GanttDayView>
+        <GanttWeekView></GanttWeekView>
+        <GanttMonthView></GanttMonthView>
+        <GanttYearView></GanttYearView>
+    </GanttViews>
+    <GanttColumns>
+        <GanttColumn Field="@nameof(FlatModel.Title)" ShowColumnMenu="false">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.PercentComplete)" VisibleInColumnChooser="false">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.Start)">
+        </GanttColumn>
+        <GanttColumn Field="@nameof(FlatModel.End)">
+        </GanttColumn>
+    </GanttColumns>
+</TelerikGantt>
+
+@code {
+    public DateTime SelectedDate { get; set; } = new DateTime(2019, 11, 11, 6, 0, 0);
+
+    class FlatModel
+    {
+        public int Id { get; set; }
+        public int? ParentId { get; set; }
+        public string Title { get; set; }
+        public double PercentComplete { get; set; }
+        public DateTime Start { get; set; }
+        public DateTime End { get; set; }
+    }
+
+    public int LastId { get; set; } = 1;
+    List<FlatModel> Data { get; set; }
+
+    protected override void OnInitialized()
+    {
+        Data = new List<FlatModel>();
+        var random = new Random();
+
+        for (int i = 1; i < 6; i++)
+        {
+            var newItem = new FlatModel()
+                {
+                    Id = LastId,
+                    Title = "Employee  " + i.ToString(),
+                    Start = new DateTime(2020, 12, 10 + i),
+                    End = new DateTime(2020, 12, 11 + i),
+                    PercentComplete = Math.Round(random.NextDouble(), 2)
+                };
+
+            Data.Add(newItem);
+            var parentId = LastId;
+            LastId++;
+
+            for (int j = 0; j < 5; j++)
+            {
+                Data.Add(new FlatModel()
+                    {
+                        Id = LastId,
+                        ParentId = parentId,
+                        Title = "    Employee " + i + " : " + j.ToString(),
+                        Start = new DateTime(2020, 12, 20 + j),
+                        End = new DateTime(2020, 12, 21 + i + j),
+                        PercentComplete = Math.Round(random.NextDouble(), 2)
+                    });
+
+                LastId++;
+            }
+        }
+
+        base.OnInitialized();
+    }
+}
+````
+
+## Notes
+
+* Applying settings to a Gantt column like `Lockable="false"` will take precedence over the common settings applied in the `<GanttColumnMenuSettings>` and disable the above-mentioned functionalitiy for the corresponding column.
```

Comment:
What is the "above-mentioned functionality"?

Reply:
"Above mentioned functionality" is used to describe Lockable in this case.